roles/network_plugin/contiv/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+contiv_cert_dir: "/etc/contiv/certs"
+contiv_netmaster_port: 9999
+contiv_cni_version: 1.0.1
+contiv_k8s_api_server: "https://{{ hostvars[groups['kube-master'][0]]['ansible_default_ipv4']['address'] }}:6443"
+
+# Parameters for Contiv api-proxy
+contiv_enable_api_proxy: true
+contiv_api_proxy_port: 10000
+contiv_generate_certificate: true
+
+# Forwarding mode: bridge or routing
+contiv_fwd_mode: bridge
+
+# Fabric mode: aci, aci-opflex or default
+contiv_fabric_mode: default
+
+# Dataplane interface
+contiv_vlan_interface: ""
+
+# Default network configuration
+contiv_default_subnet: "10.200.0.0/24"
+contiv_default_gw: "10.200.0.1"
```

Comment:
You can simplify this with a calculation based on above:
`{{contriv_default_subnet|ipaddr('net')|ipaddr(1)|ipaddr('address') }}`